Makefile

```
@@ -81,11 +81,11 @@ install-tools:
 	go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2.0
 	go install gotest.tools/gotestsum@v1.10.0
 	go install golang.org/x/tools/cmd/goimports@latest
+	bash ./scripts/install-golangci-lint.sh v1.55.0
 
 .PHONY: lint
 lint: fmt ## Run linters on all go files
-	docker run --rm -v $(shell pwd):/app:ro -w /app golangci/golangci-lint:v1.54.2 bash -e -c \
-		'golangci-lint run -v --timeout 5m'
+	golangci-lint run -v --timeout 5m
```

What if we do:

```suggestion
	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.55.1 run 
```

A simpler way to have multiple version of `golangci-lint` in the same system

---

We should try to always run the latest also in other projects. I don't mind the idea though, I'm good either way tbh :+1: